invadedhouse/commands/help.py

```diff
@@ -0,0 +1,47 @@
+from commands import Command
+
+from models import House
+from models import Player
+
+
+class Help(Command):
+
+    def __init__(
+        self, name: str,
+        player: Player, house: House
+    ):
+        super().__init__(name, player, house)
+        self.commands = []
+        self.keys_alpha = []
+        self.map = {}
+
+    def set_all_commands(self, commands: [Command]):
+        keys = []
+        cmd_idx = {}
+        for cmd in commands:
+            if not cmd.is_secret():
+                keys.append(cmd.get_name())
+            cmd_idx.update({
+                cmd.get_name(): cmd
+            })
+        self.keys_alpha = sorted(keys)
+        self.map = cmd_idx
+
+    def do(self, text: str):
+        for key in self.keys_alpha:
+            print(f'{key} - {self.map[key].get_description()}')
```

Comment:
isn't it super cool, once you've set things up nicely, that spitting out the help command is just two lines of code?

Reply:
YEAH!